src/components/marketingTab/MarketingTab.tsx

```diff
@@ -19,7 +19,7 @@ interface MarketingTabProps extends Omit<BoxProps, 'ref'> {
 
 const MarketingTab: GenericComponent<MarketingTabProps> = forwardRef(
```

Comment:
type `forwardRef` with `<HtmlElement, MarketingTabProps>` and then you also have to type `useImperativeRef` with `<HtmlElement | null, HtmlElement | null>`